src/os/ui/filter/op/isfalseop.js

```diff
@@ -0,0 +1,62 @@
+goog.provide('os.ui.filter.op.IsFalse');
+
+goog.require('os.ui.filter.op.Op');
+
+
+/**
+ * A 'PropertyIsFalse' operation class.
+ *
+ * @extends {os.ui.filter.op.Op}
+ * @constructor
+ */
+os.ui.filter.op.IsFalse = function() {
+  os.ui.filter.op.IsFalse.base(this, 'constructor',
+      'And', 'is false', 'false', undefined, undefined, 'Supports false, 0, and "false"', 'span', true);
+  this.matchHint = 'false.operation';
+};
+goog.inherits(os.ui.filter.op.IsFalse, os.ui.filter.op.Op);
+
+/**
+ * Because OpenSphere allows isEmpty() comparator for Boolean type properties, fail null/empty tests
+ *
+ * @inheritDoc
+ */
+os.ui.filter.op.IsFalse.prototype.getEvalExpression = function(v, literal) {
+  return '(!(typeof ' + v + '==="undefined"||' + v + '==null||' + v + '.length==0)&&(' +
+      v + '===false||' + v + '==0||(""+' + v + ').toLowerCase()=="false"))';
+};
+
+/**
+ *
+ * @inheritDoc
+ */
+os.ui.filter.op.IsFalse.prototype.getFilter = function(column, literal) {
+  var f = [];
+
+  f.push('<' + this.localName + '>');
```

Comment:
After discussion with @wallw-bits -- Determined that I need the matchHint, etc attributes in the root XML tag so the filter builder can parse back to "falseop" instead of complex "and...or...etc"